compatibility-v1.php

```diff
@@ -42,14 +42,14 @@ function json_v1_compatible_routes( $routes ) {
 }
 
 /**
- * Use Reflection to match request parameters to function parameters
+ * Use Reflection to match request parameters to function parameters.
  *
- * @param mixed $result Result to use
- * @param WP_JSON_Request $request Request object
- * @return mixed
+ * @param mixed $result Result to use.
+ * @param WP_JSON_Request $request Request object.
+ * @return mixed.
```

Comment:
Hey, @danielbachhuber !

Periods at the end of the type are the part of standard WordPress PHP documentation in the Core handbook. You can check this information here https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods